colp_build/sql/name.sql

```diff
@@ -0,0 +1,110 @@
+		                                REGEXP_REPLACE(
+		                                    REGEXP_REPLACE(
+		                                        REGEXP_REPLACE(
+		                                            REGEXP_REPLACE(
+		                                                REGEXP_REPLACE(
+		                                                    REGEXP_REPLACE(
+		                                                        REGEXP_REPLACE(
+		                                                            REGEXP_REPLACE(
+		                                                                REGEXP_REPLACE(
+		                                                                    REGEXP_REPLACE(
+		                                                                        REGEXP_REPLACE(
+		                                                                            REGEXP_REPLACE(
+		                                                                                REGEXP_REPLACE(
+		                                                                                    REGEXP_REPLACE(
+		                                                                                        REGEXP_REPLACE(
+		                                                                                            REGEXP_REPLACE(TRIM(parcel_name)||' ', 
+		                                                                                            ' & ', 
+		                                                                                            ' AND '
+		                                                                                            ), 
+		                                                                                        'PLAYLOT|PLGD|PLAYGRD|PLAYGND|PLYGRND|PLYGD|PLAYGRND|PLYGRD|PLGRD|PLGD.|PLYGD|PLAYGD', 
+		                                                                                        'PLAYGROUND'
+		                                                                                        ),
+		                                                                                    '@',
+		                                                                                    'AT'
+		                                                                                    ),
+		                                                                                'HSES',
+		                                                                                'HOUSES'
+		                                                                                ),
+		                                                                            ' \(GROUP 1 ',
+		                                                                            ' (GROUP 1) '
+		                                                                            ),
+		                                                                        ' PK',
+		                                                                        ' PARK'
```

Comment:
Maybe this doesn't work because 'ANNANDALE STATION PKG' is becoming 'ANNANDALE STATION PARKG' and 'BRONX & PELHAM PKWY' is now 'BRONX AND PELHAM PARKWY'
You can change it so that only instances where it's '%PK' or '%PK %' get changed to 'PARK' or remove the logic